src/gl/pointFeature.js

```diff
@@ -26,6 +26,7 @@ geo.gl.pointFeature = function (arg) {
       m_pixelWidthUniform = null,
       m_aspectUniform = null,
       m_dynamicDraw = arg.dynamicDraw === undefined ? false : arg.dynamicDraw,
+      m_primitiveShape = arg.primitiveShape || "triangle",
```

Comment:
I am okay if we are just checking for triangle argument only 

if (arg.primitiveShape && (arg.primitiveShape == "square" || ... "triangle") {
} else {
// default to triangle
}